packages/pn-personafisica-webapp/src/components/DomicileBanner/DomicileBanner.tsx

```diff
@@ -108,43 +107,47 @@ const DomicileBanner: React.FC<Props> = ({ source }) => {
   const dispatch = useAppDispatch();
   const open = useAppSelector((state: RootState) => state.generalInfoState.domicileBannerOpened);
   const { IS_DOD_ENABLED } = getConfiguration();
+  const {
+    defaultPECAddress,
+    defaultSERCQ_SENDAddress,
+    defaultAPPIOAddress,
+    defaultEMAILAddress,
+    defaultSMSAddress,
+  } = useAppSelector(contactsSelectors.selectAddresses);
 
-  const digitalAddresses = useAppSelector(
-    (state: RootState) => state.contactsState.digitalAddresses
-  );
-
-  const hasSercqSend = digitalAddresses.find((addr) => addr.channelType === ChannelType.SERCQ_SEND);
-  const hasAppIODisabled = digitalAddresses.find(
-    (addr) => addr.channelType === ChannelType.IOMSG && addr.value === IOAllowedValues.DISABLED
-  );
+  const hasAppIODisabled = defaultAPPIOAddress?.value === IOAllowedValues.DISABLED;
 
-  const hasCourtesyAddresses =
-    digitalAddresses.filter(
-      (addr) => addr.addressType === AddressType.COURTESY && addr.value !== IOAllowedValues.DISABLED
-    ).length > 0;
-  const domicileBannerData: DomicileBannerData | null = getDomicileData(
-    source,
-    !!hasSercqSend,
-    hasCourtesyAddresses,
-    !!hasAppIODisabled,
-    IS_DOD_ENABLED
-  );
+  const hasCourtesyAddresses = !!(defaultEMAILAddress || defaultSMSAddress);
+  const domicileBannerData: DomicileBannerData | null = defaultPECAddress
+    ? null
+    : getDomicileData(
+        source,
+        !!defaultSERCQ_SENDAddress,
+        hasCourtesyAddresses,
+        hasAppIODisabled,
+        IS_DOD_ENABLED
+      );
 
   const handleClose = () => {
     dispatch(closeDomicileBanner());
-    // sessionStorage.setItem('domicileBannerClosed', 'true');
+    sessionStorage.setItem('domicileBannerClosed', 'true');
   };
 
   const handleClick = (destination?: ChannelType, operation?: ContactOperation) => {
     if (destination && operation) {
-      dispatch(setExternalEvent({ destination, source, operation }));
+      if (destination === ChannelType.SERCQ_SEND && operation === ContactOperation.ADD) {
+        navigate(routes.DIGITAL_DOMICILE_ACTIVATION);
+        return;
+      } else {
+        dispatch(setExternalEvent({ destination, source, operation }));
```

Comment:
this should dispatched anyway because it is used to trace the source from where the dod is enabled